cmd/genji/shell/command.go

```diff
@@ -136,3 +141,193 @@ func displaySuggestions(in string) error {
 	fmt.Println()
 	return nil
 }
+
+func getDatabaseTransaction(db *genji.DB) (*database.Transaction, error) {
+	var dtx *database.Transaction
+	var err error
+	err = db.View(func(tx *genji.Tx) error {
```

Comment:
This code does this:

- Use DB to create a transaction A
- Use transaction A to get DB back
- Use DB to create transaction B
- Close transaction A

Why not just use `db.Begin(false)`?

Reply:
In fact `tx *genji.Tx` is sufficient.